orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ResolveDeploySourceManifestTask.java

```diff
@@ -54,4 +58,21 @@ public TaskResult execute(@Nonnull StageExecution stage) {
         .put("optionalArtifacts", result.getOptionalArtifacts())
         .build();
   }
+
+  // DeployManifestContext only accepts List<Map<Object, Object>>,
+  // Spel is not flattening nested List anymore
+  // We flatten them because its possible have a List of manifests resolved by spel
+  private List<Object> flattenNestedLists(Object manifests) {
+    List<Object> result = new ArrayList<>();
+    List<Object> tmp = (List<Object>) manifests;
+
+    for (Object obj : tmp) {
```

Comment:
Just a heads up there is an NPE here when manifests is null - fix in: https://github.com/spinnaker/orca/pull/4044

Reply:
thanks for fixing this, I will review your PR